composer.json

```diff
@@ -69,6 +71,13 @@
       "plugins/toolbox/tools/imagemanipulation/class.tx_dlf_toolsImagemanipulation.php",
       "plugins/toolbox/tools/imagedownload/class.tx_dlf_toolsImagedownload.php",
       "plugins/validator/class.tx_dlf_validator.php"
+    ],
+    "psr-4" : {
+      "Solarium\\" : "lib/solarium/src",
```

Comment:
If we don't ship Solarium with the extension, but inject it as a dependency via composer, how does this work then? Shouldn't this be something like `vendor/solarium/...`? Or, alternatively, shouldn't there an option like `"installer-paths" : { "lib/solarium/src" : ["solarium/solarium"] }`?

Reply:
Pinging @albig: What would be the prefered way to do it?

Reply:
I had simply overlooked this code within the remove commit. If we rely on composer, everything should work fine without the psr-4 entries here.

Reply:
Fixed